graspologic/cluster/divisive_cluster.py

```diff
@@ -238,26 +265,68 @@ def _fit(self, X):
                             (labels, np.zeros((len(X), 1), dtype=int))
                         )
                     labels[inds, 1 : child_labels.shape[1] + 1] = child_labels
+                else:
+                    # make a sudo "GaussianMixture" model for the cluster
+                    # that was not fitted
+                    if self.cluster_method == "gmm":
+                        cluster_indx = len(dc.parent.children) - 1
+                        parent_model = dc.parent.model_
+                        # for cases where "init_params" is "kmeans" in AutoGMM
+                        model = {
+                            "covariance_type": parent_model.covariance_type,
+                            "reg_covar": parent_model.reg_covar,
+                        }
+                        if parent_model.means_init is not None:
+                            model.update(
+                                {
+                                    "means_init": parent_model.means_init[cluster_indx],
+                                    "precisions_init": parent_model.precisions_init[
+                                        cluster_indx
+                                    ],
+                                    "weights_init": np.array([1]),
+                                }
+                            )
```

Comment:
also from our conversation wednesday - I felt like we should either decide not to have this approach or to create a true GaussianMixture object for each node in the clustering tree. I don't feel super strongly which and happy to discuss more, I did feel that what we have here right now is kinda halfway inbetween

Reply:
i am ready to merge besides this point - I still feel like what is in this PR is a half measure between an actual fitted GM model and nothing, so right now I'm thinking we just omit in this PR until we decide something else. 

I think the .model attribute is only useful if it has a fitted mean and covariance, is why I'm still uncertain about this. currently im not sure why we'd need the mean_init and precision_init, for instance